### PR TITLE
improved tracy bar panel template

### PR DESCRIPTION
**Pull request by petrparolek**
Before

obrazek

After my improvement

obrazek

Panel looks lie better in my opinion, it is inspired by Kdyby/Doctrine tracy bar panel.

### CODE REVIEW

File: src/Tracy/QueryPanel/templates/panel.phtml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <?php
-
 /** @var mixed[] $parameters */
 /** @var object[] $queries */
 /** @var int $queriesNum */
-/** @var float $totalTime */
 
+/** @var float $totalTime */
```
Comment (mabar) [inline src/Tracy/QueryPanel/templates/panel.phtml]: Why that formating change
```
<?php
/** @var mixed[] $parameters */
/** @var object[] $queries */
/** @var int $queriesNum */

/** @var float $totalTime */
use Nettrine\DBAL\Utils\QueryUtils;
```

**Comment by mabar** [comment]
Please, don't create multiple PRs for same thing (#40)

File: src/Tracy/QueryPanel/templates/panel.phtml (hunk 1 above)
Comment (petrparolek) [reply src/Tracy/QueryPanel/templates/panel.phtml]: I can fix it- I have turned on auto format code in Netbeans

**Comment by petrparolek** [comment]
sorry, měl jsem překlep v commitu a v branch

**Comment by mabar** [comment]
`git rebase -i HEAD~n` (`n` is number of commits to edit) to edit commits history. `git push origin HEAD --force` to push it to remote. Source branch can be changed at Github.
Also, keep Github conversations English only, please

File: src/Tracy/QueryPanel/templates/panel.phtml (hunk 1 above)
Comment (petrparolek) [reply src/Tracy/QueryPanel/templates/panel.phtml]: fixed

**Comment by petrparolek** [comment]
tank you for tips

**Comment by vody105** [comment]
Hi, It's great that you are interested in the panel.

I like the addition `time: X ms`  in the heading.

On the other hand:
 - `ms` on every row is pointless IMHO and worsen readability
- `source` toggle is usefull because unlike Kdyby we list whole backtrace in your application (it can get messy really fast). Try to call your queries from inside your model or control classes.

Have fun,
Filip

**Pull request closed without merging**